Example/Tests/GINIComponentAPICoordinator.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  GINIComponentAPICoordinator.swift
+//  GiniVision_Tests
+//
+//  Created by Enrique del Pozo Gómez on 11/13/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVision_Example
+@testable import GiniVision
+
+class GINIComponentAPICoordinator: XCTestCase {
```

Comment:
Only two tests for so many changes???

Reply:
I forgot to tell you that I had not finished.